chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/G3Sed2015DownloadableResourceCreatorJob.java

```diff
@@ -128,8 +128,8 @@ public void execute(JobExecutionContext jobContext) throws JobExecutionException
     private void writeToFile(File downloadFolder, List<CertifiedProductSearchDetails> results) throws IOException {
         String csvFilename = downloadFolder.getAbsolutePath()
                 + File.separator
-                + "chpl-sed-all-details-"
-                + getFilenameTimestampFormat().format(new Date())
+                + env.getProperty("SEDDownloadName")
```

Comment:
Did this property already exist, yet wasn't being used?

Reply:
Yes. The property was being used where the file was read/downloaded but not here where the file is written.